modules/apps/comment/comment-test/src/testIntegration/java/com/liferay/comment/subscriptions/test/CommentsSubscriptionTest.java

```diff
@@ -101,6 +101,8 @@ public void testCannotSubscribeToCommentsWithoutViewPermissions()
 
 		Assert.assertFalse(
 			discussionPermission.hasSubscribePermission(
+				PermissionCheckerFactoryUtil.create(
+					UserLocalServiceUtil.getGuestUser(_group.getCompanyId())),
 				TestPropsValues.getCompanyId(), _group.getGroupId(),
```

Comment:
@W2jx, in this commit (https://github.com/liferay-core-infra/liferay-portal/blob/593c08b4f414dc5bbcddb96211aec620ad9d1430/modules/apps/comment/comment-test/src/testIntegration/java/com/liferay/comment/subscriptions/test/CommentsSubscriptionTest.java#L97-L107)

`PermissionCheckerFactoryUtil.create(UserLocalServiceUtil.getGuestUser(_group.getCompanyId())` was used twice in the same method,  Please extract one param for it.